cmake-ex/Dockerfile

```
@@ -0,0 +1,27 @@
+#!/bin/bash
+FROM ubuntu:20.04
+
+# Copy files to Docker container
+RUN mkdir cmake-exercise
+COPY cmake-ex/ /cmake-exercise/
```

If you run the Dockerfile from `cmake-ex`, this does not work.

```suggestion
ADD . /cmake-exercise
```

---

Actually, I didn't knew this "trick" when working on this exercise :see_no_evil:. In later exercises, I used this and am very happy to know how to avoid such copying :smiley:.  
The only purpose of this was to get the files into the Docker container.